### PR TITLE
Button-tags

### LINKED ISSUE #103: Button - tags
tags

### PR DESCRIPTION
**Pull request by jminguely**
- Creation of a content (atom) **Badges**
- Application of these style to the component **Button-tag**

Closes #103

### CODE REVIEW

**Comment by jminguely** [comment]
Done !

I've included selectize.js. I'm not sure about the hierarchy thou. I created a variant from the tags with this selectize component. Should I create an atom or molecule named Selectize / Input tags / Tags selector ?

File: assets/components/atoms/tag/tag-input.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      }
+    },
+    create: function(input) {
+        return {
+            value: input,
+            text: input
+        }
+    }
+  });
+};
```
Comment (Yago) [inline assets/components/atoms/tag/tag-input.js]: Can be more ES2015, like : 

```js
export default () => {
  $('.selectize').selectize({
    plugins: ['remove_button'],
    render: {
      item: (data, escape) => `<div class="badge tag tag-primary">${escape(data.text)}</div>`,
    },
    create: input => ({
      value: input,
      text: input,
    }),
  });
};
```

File: assets/components/atoms/tag/tag.scss [hunk 2]
```diff
@@ -0,0 +1,116 @@
+@charset 'utf-8';
+
+a.tag {
+  &:hover {
+    background: $gray-100;
+    border-top-color: $gray-500;
```
Comment (Yago) [inline assets/components/atoms/tag/tag.scss]: `$gray-500` doesn't exist anymore 😅

Comment (Yago) [reply assets/components/atoms/tag/tag.scss]: Same for all the others gray except for 100, 200, 300 & 600

File: assets/components/organisms/button/button-tags.twig [hunk 3]
```diff
@@ -1 +1,35 @@
...
+    <span class="badge tag tag-primary">Turbo</span>
+    <span class="badge tag tag-primary">Algues vertes</span>
+    <span class="badge tag tag-primary">Beurre demi-sel</span>
+    <span class="badge tag tag-primary">Saumon</span>
+  </div>
+  <div class="col-md-6">
+    <p class="mb-1 mt-3">Exemple d'utilisation</p>
+    <div class="form-group">
+      <label class="custom-control-label" for="">Vos critères de recherches</label>
+      <input type="text" class="selectize">
```
Comment (Yago) [inline assets/components/organisms/button/button-tags.twig]: Maybe you can add some default value (`value="truc, machin"`) 😉

File: assets/components/atoms/tag/tag.twig [hunk 4]
```diff
@@ -0,0 +1,19 @@
+<h5>non-link</h5>
+<span class="badge tag tag-primary">Primary</span>
```
Comment (Yago) [inline assets/components/atoms/tag/tag.twig]: Is `.badge` usefull ? Can we get rid of it ?

**Comment by Yago** [comment]
Except for `.badge`, the naming and the hierarchy is perfect 😉

**Comment by jminguely** [comment]
Thanks for the review. It's should be okay now. The FF issue was fixed when removing the now useless bootstrap `.badge` class

**Comment by Yago** [comment]
yes

**Pull request merged**